verification/dependencies/fmt/fmt.gobra

```diff
@@ -22,8 +22,12 @@ preserves forall i int :: { &v[i] } 0 <= i && i < len(v) ==> acc(&v[i], R55)
 // TODO:
 // The following precondition cannot be adequately captured in Gobra.
 // preserves forall i int :: 0 <= i && i < len(v) ==> definitions.IsOfPrimitiveType(v[i])
+// TODO: Once Gobra issue #846 is resolved, capture the sensitivity of `v`
+// using a pure Boolean function.
```

Comment:
how do you plan on doing that? by changing the input type to an interface that contains such a ghost function? Whatever decision we make, we should make the same decision for `serrors.New`